Sources/TuistGenerator/Generator/ProjectDescriptorGenerator.swift

```diff
@@ -124,11 +149,13 @@ final class ProjectDescriptorGenerator: ProjectDescriptorGenerating {
             name: project.xcodeProjPath.basename
         )
 
-        let schemes = try schemeDescriptorsGenerator.generateProjectSchemes(
-            project: project,
-            generatedProject: generatedProject,
-            graphTraverser: graphTraverser
-        )
+        let schemes = try Signpost.measure(category: "ProjectDescriptorGenerator", identifier: "generateProjectSchemes", label: project.name) {
+            try schemeDescriptorsGenerator.generateProjectSchemes(
+                project: project,
+                generatedProject: generatedProject,
+                graphTraverser: graphTraverser
+            )
+        }
```

Comment:
Up to you but I think I'd break the arguments into lines for readability. Like this or otherwise:

```suggestion
        let schemes = try Signpost.measure(
            category: "ProjectDescriptorGenerator",
            identifier: "generateProjectSchemes",
            label: project.name
        ) {
            try schemeDescriptorsGenerator.generateProjectSchemes(
                project: project,
                generatedProject: generatedProject,
                graphTraverser: graphTraverser
            )
        }
```

Reply:
ah yes +1 to separating the parameters for readability - this wasn't an intentional style choice mostly a result of quick prototyping